pages/index.tsx

```diff
@@ -222,13 +239,23 @@ export async function getServerSideProps(context: any) {
     donationStep = 1;
     try {
       const requestParams = {
-        url: `/app/projects/${to}`,
+        url: `/app/projects/${to}/paymentOptions?country=${country}`,
         setshowErrorCard,
         tenant,
       };
-      const project = await apiRequest(requestParams);
-      if (project.data) {
-        projectDetails = project.data;
+      const paymentOptionsResponse = await apiRequest(requestParams);
+      if (paymentOptionsResponse.data) {
+        projectDetails = {
+          id: paymentOptionsResponse.data.id,
+          name: paymentOptionsResponse.data.name,
+          description: paymentOptionsResponse.data.description,
+          purpose: paymentOptionsResponse.data.purpose,
+          ownerName: paymentOptionsResponse.data.ownerName,
+          taxDeductionCountries:
+            paymentOptionsResponse.data.taxDeductionCountries,
+          ownerImage: paymentOptionsResponse.data.image,
```

Comment:
this is not `ownerImage`, this is `projectImage`

Reply:
@tejassonar

Reply:
Done